content/membership/join-us/_index.md

```diff
@@ -5,4 +5,4 @@ hide_sidebar: true
 jumbotron_bg_class: "featured-jumbotron-sub-pages header-membership-bg-img"
 ---
 
-{{< page/join-us >}}
+{{< hubspot_contact_form portalId="5413615" formId="b2d1d804-1d76-4fe9-9647-03f478af1780" >}}
```

Comment:
I'm assuming you're adding this here since you're making changes to ECDtools?

Reply:
We should be using our shortcode for this.

Reply:
I don't see any conflicts?